tests/utils_test.py

```diff
@@ -0,0 +1,60 @@
+import datetime
+import pytest
+import numpy as np
+import becquerel as bq
+
+
+# ----------------------------------------------
+#         Test utils
+# ----------------------------------------------
+
+
+def test_sqrt_bins():
+    """Test basic functionality of utils.sqrt_bins."""
+    edge_min = 0
+    edge_max = 3000
+    n_bins = 128
+    be = bq.utils.sqrt_bins(edge_min, edge_max, n_bins)
+    bc = (be[1:] + be[:-1]) / 2
+    bw = np.diff(be)
+    # compute slope of line
+    m = np.diff(bw**2) / np.diff(bc)
+    # assert that the square of the bin
+    assert np.allclose(m[0], m)
+    # negative edge_min
+    with pytest.raises(AssertionError):
+        be = bq.utils.sqrt_bins(-10, edge_max, n_bins)
+    # edge_max < edge_min
+    with pytest.raises(AssertionError):
+        be = bq.utils.sqrt_bins(100, 50, n_bins)
+
+
+@pytest.mark.parametrize(
+    "timestamp",
+    [
+        datetime.date(year=2023, month=6, day=14),
+        datetime.datetime(year=2023, month=6, day=14, hour=0, minute=0, second=0),
+        "2023_06_14_00_00_00",
```

Comment:
Can you also include an ISO 8601 string, like "2023-06-14T00:00:00.000Z-0000"? That one would go through `dateutils` but not `strptime`.

Reply:
Time zone might mess things up since the others aren't timezone-aware, so it probably needs to be omitted.

Reply:
Good call, yeah. I just dropped it in the check but not in the input.